crm_stage_multi_company/security/crm_security.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="crm_stage_company_rule" model="ir.rule">
+        <field name="name">CRM Stage Multi-Company</field>
+        <field name="model_id" ref="crm.model_crm_stage" />
+        <field
+            name="domain_force"
+        >['|',('company_id','=',False),('company_id','child_of',[user.company_id.id])]</field>
```

Comment:
Since https://github.com/odoo/odoo/commit/a04df5cb1411a14752ba7a4d6b3c4b7b4f1d6d44
these rules are written as follow in Odoo :
```suggestion
        >[('company_id', 'in', company_ids + [False])]</field>
```

Maybe it should be better to follow this guideline ?